GiniVision/Classes/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -225,6 +241,12 @@ extension GiniScreenAPICoordinator {
     }
     
     @objc func showAnalysisScreen() {
+        
+        if screenAPINavigationController.topViewController is MultipageReviewViewController ||
+        screenAPINavigationController.topViewController is ReviewViewController {
+            trackingDelegate?.onReviewScreenEvent(event: Event(type: .next))
```

Comment:
`.next` could go into the `ReviewViewController` and then it's also triggered when using the Component API.

Reply:
I'm not sure you can. The Next button is managed by the coordinator.

Reply:
I see, of course, the next button is in the navigation bar, forgot about that.